src/components/publishing/header/feature_header.tsx

```diff
@@ -64,7 +64,7 @@ function renderTextLayoutAsset(url, layout, title, children) {
       const image = <Image src={src} alt={alt} />
       return (
         <TextAsset>
-          {children[2]}
+          {children[3]}
```

Comment:
More of a readability thing, but at the top of the render where we pull out children in the first place, it might be helpful to de-mystify what these children are by naming them before passing them into other places.